website/app/initializers/showdown-extensions.js

```diff
@@ -58,6 +58,7 @@ export function initialize(/* application */) {
         var end = (options.omitExtraWLInCodeBlocks) ? '' : '\n';
 
         let codeblock = inputCodeblock;
+        let codeblockcopy =  encodeURI(inputCodeblock);
```

Comment:
nitpicking
- maybe add a comment explaining why we need to encode the content (same where we use it in the encoded form)
- maybe use the word `encoded` in the variable name (eg. `codeblockEncoded`)